packages/vsce/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable changes to the "cics-extension-for-zowe" extension will be documente
 
 ## Recent Changes
 
+Enhancement: Display a clear "No regions found in plex" message when a plex contains no managed regions. [#444] (https://github.com/zowe/cics-for-zowe-client/issues/444)
```

Comment:
The hyperlink won't render properly with a space - I know there are others in there that i've fixed in my branch so just update this one.

Also I think the issue was created as a bug rather than a feature request.

```suggestion
- Bugfix: Display a clear message when a plex contains no managed regions. [#444](https://github.com/zowe/cics-for-zowe-client/issues/444)
```